app/src/lib/editors/win32.ts

```diff
@@ -218,6 +218,18 @@ function getRegistryKeys(
       ]
     case ExternalEditor.SlickEdit:
       return [
+        // 64-bit version of SlickEdit
+        {
+          key: HKEY.HKEY_LOCAL_MACHINE,
+          subKey:
+            'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\vs.exe',
```

Comment:
Hey @texasaggie97, would you mind elaborating on why the `Uninstall` path approach doesn't work for SlickEdit 2019? I don't know anything about the editor I'm afraid and it makes me a little bit nervous when we're diverging from an established pattern.

Reply:
The uninstall path works fine. I just don't want to have to go through this every time there is a new release and add the new GUIDs. This is my attempt at making this work for future releases as well.

Reply:
I definitely understand the frustration of having to update for each release but I'm not comfortable with diverging from the convention in favor of an app path entry. Had the executable's name been something more distinctive like `VisualSlickEdit.exe` or similar I think I may have felt more comfortable.

The whole editor resolution part of Github Desktop is due for an overhaul and I'd love it if we could find a more flexible approach than looking up these uninstall keys but we're a small team of engineers working on Github Desktop and consistency helps us have confidence in these changes where we're not able to install and test every editor out ourselves.

Would you mind changing this back to the uninstall key?